app/lib/import_export_constants.rb

```diff
@@ -38,32 +38,36 @@ module ImportExportConstants # rubocop:todo Metrics/ModuleLength
+                              :primary_telephone_type, :secondary_telephone, :secondary_telephone_type, :preferred_contact_time, :email, :port_of_origin,
+                              :date_of_departure, :source_of_report, :flight_or_vessel_number, :flight_or_vessel_carrier, :port_of_entry_into_usa,
+                              :date_of_arrival, :travel_related_notes, :additional_planned_travel_type, :additional_planned_travel_destination,
+                              :additional_planned_travel_destination_state, :additional_planned_travel_destination_country,
+                              :additional_planned_travel_port_of_departure, :additional_planned_travel_start_date, :additional_planned_travel_end_date,
+                              :additional_planned_travel_related_notes, :last_date_of_exposure, :potential_exposure_location, :potential_exposure_country,
+                              :contact_of_known_case, :contact_of_known_case_id, :travel_to_affected_country_or_area,
+                              :was_in_health_care_facility_with_known_cases, :was_in_health_care_facility_with_known_cases_facility_name, :laboratory_personnel,
+                              :laboratory_personnel_facility_name, :healthcare_personnel, :healthcare_personnel_facility_name,
+                              :crew_on_passenger_or_cargo_flight, :member_of_a_common_exposure_cohort, nil, :exposure_risk_assessment, :monitoring_plan,
+                              :exposure_notes, :full_status, :symptom_onset, :case_status, :lab_1_type, :lab_1_specimen_collection, :lab_1_report,
+                              :lab_1_result, :lab_2_type, :lab_2_specimen_collection, :lab_2_report, :lab_2_result, :jurisdiction_path, :assigned_user,
+                              :gender_identity, :sexual_orientation, :race_other, :race_unknown, :race_refused_to_answer, :vaccine_1_group_name,
+                              :vaccine_1_product_name, :vaccine_1_administration_date, :vaccine_1_dose_number, :vaccine_1_notes, :vaccine_2_group_name,
+                              :vaccine_2_product_name, :vaccine_2_administration_date, :vaccine_2_dose_number, :vaccine_2_notes, :follow_up_reason,
+                              :follow_up_note, :vaccine_3_group_name, :vaccine_3_product_name, :vaccine_3_administration_date, :vaccine_3_dose_number,
+                              :vaccine_3_notes, :international_telephone, :contact_type, :contact_name, :alternate_contact_type, :alternate_contact_name,
+                              :alternate_preferred_contact_method, :alternate_preferred_contact_time, :alternate_primary_telephone,
+                              :alternate_primary_telephone_type, :alternate_secondary_telephone, :alternate_secondary_telephone_type,
+                              :alternate_international_telephone, :alternate_email, :cohort_1_type, :cohort_1_name, :cohort_1_location, :cohort_2_type,
+                              :cohort_2_name, :cohort_2_location].freeze
```

Comment:
Can't comment on the excel file, so commenting somewhere. I am wondering if we want to make it a little more obvious in the excel template that these two fields:
![image](https://user-images.githubusercontent.com/52747882/142059660-ed9dc36a-0016-4f93-aecf-673c0df666c7.png)

are ignored. I would think either:
1. Add some color coding to the template and the guidance, maybe like a red or something to indicate these fields are ignored.
2. The more extreme measure would be to just log an error if these fields have a value. Personally I'm for this, since then there is no way someone will think they are getting data into Sara Alert without realizing we are ignoring it. But we may want to clear this with the PO team.

Reply:
I went with #1 for now: https://github.com/SaraAlert/SaraAlert/pull/1125/commits/237545888b608d51912da918ac3e3e6d0780ee28#